graph/pull_v2.go

```diff
@@ -193,9 +191,8 @@ func (p *v2Puller) download(di *downloadInfo) {
 	di.err <- nil
 }
 
-func (p *v2Puller) pullV2Tag(tag, taggedName string) (verified bool, err error) {
+func (p *v2Puller) pullV2Tag(out io.Writer, tag, taggedName string) (verified bool, err error) {
```

Comment:
I added this argument so that `pullV2Tag` can write to the `ProgressStatus` writer, not its usual writer.